h2o-automl/src/main/java/water/rapids/ast/prims/mungers/AstTargetEncoderTransform.java

```diff
@@ -65,6 +66,15 @@ public ValFrame apply(Env env, Env.StackHelp stk, AstRoot asts[]) {
     }
   }
 
+  private BlendingParams getBlendingParams(Env env, Env.StackHelp stk, AstRoot[] asts, boolean withBlending) {
```

Comment:
I think it is better not to call the method if it is not supposed to do anything. This is an unusual pattern.

Reply:
Agree